[2022-09-08T18:49:18.443+0500] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: expectation_dag.expect_exist manual__2022-09-08T13:49:05.304007+00:00 [queued]>
[2022-09-08T18:49:18.448+0500] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: expectation_dag.expect_exist manual__2022-09-08T13:49:05.304007+00:00 [queued]>
[2022-09-08T18:49:18.448+0500] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-08T18:49:18.448+0500] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2022-09-08T18:49:18.448+0500] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-08T18:49:18.462+0500] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): expect_exist> on 2022-09-08 13:49:05.304007+00:00
[2022-09-08T18:49:18.466+0500] {standard_task_runner.py:52} INFO - Started process 1069660 to run task
[2022-09-08T18:49:18.468+0500] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'expectation_dag', 'expect_exist', 'manual__2022-09-08T13:49:05.304007+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/expectation_dag.py', '--cfg-path', '/tmp/tmp1qcr68ks', '--error-file', '/tmp/tmpgl2z16ex']
[2022-09-08T18:49:18.468+0500] {standard_task_runner.py:80} INFO - Job 30: Subtask expect_exist
[2022-09-08T18:49:18.526+0500] {task_command.py:371} INFO - Running <TaskInstance: expectation_dag.expect_exist manual__2022-09-08T13:49:05.304007+00:00 [running]> on host shahroze
[2022-09-08T18:49:18.571+0500] {taskinstance.py:1581} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=expectation_dag
AIRFLOW_CTX_TASK_ID=expect_exist
AIRFLOW_CTX_EXECUTION_DATE=2022-09-08T13:49:05.304007+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-08T13:49:05.304007+00:00
[2022-09-08T18:49:18.573+0500] {logging_mixin.py:115} INFO - {
  "success": true,
  "result": {},
  "meta": {},
  "expectation_config": {
    "meta": {},
    "kwargs": {
      "column": "trip_distance",
      "result_format": "BASIC"
    },
    "expectation_type": "expect_column_to_exist"
  },
  "exception_info": {
    "raised_exception": false,
    "exception_traceback": null,
    "exception_message": null
  }
}
[2022-09-08T18:49:18.573+0500] {python.py:173} INFO - Done. Returned value was: None
[2022-09-08T18:49:18.578+0500] {taskinstance.py:1407} INFO - Marking task as SUCCESS. dag_id=expectation_dag, task_id=expect_exist, execution_date=20220908T134905, start_date=20220908T134918, end_date=20220908T134918
[2022-09-08T18:49:18.600+0500] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-08T18:49:18.612+0500] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
